README.md

```diff
@@ -2,47 +2,60 @@
 
 The purpose of this service is to convert given arguments into a prompt, use an LLM and send the result back.
 
-The service will receive queries from api.serlo.org which should receive them via GraphQL operation from frontend. 
+The service will receive queries from api.serlo.org which should receive them via GraphQL operation from frontend.
 
 The way through api.serlo.org is to check if a user is logged in and has the role required for the feature.
 
-## Development
+## Development locally
 
-* Install the Python version in [.tool-versions](.tool-versions)
-    * You may use [asdf](https://asdf-vm.com/) for the installation.
-* Install the requirements using [pipenv](https://pipenv.pypa.io/en/latest/installation/#installing-pipenv)
-* Run `pipenv shell` to activate the project's [virtual environment](https://docs.python.org/3/library/venv.html). 
-* Run `pipenv install --dev` to install the development dependencies.
-* Run `pipenv run lint` to run the linting.
-* Run `pipenv run format` to format the code.
-* Run `pipenv run type_check` to run the static type checker ([mypy](https://github.com/python/mypy)).
+- Install the Python version in [.tool-versions](.tool-versions)
+  - You may use [asdf](https://asdf-vm.com/) for the installation.
+- Install the requirements using [pipenv](https://pipenv.pypa.io/en/latest/installation/#installing-pipenv)
+- Run `pipenv shell` to activate the project's [virtual environment](https://docs.python.org/3/library/venv.html).
+- Run `pipenv install --dev` to install the development dependencies.
+- Run `pipenv run lint` to run the linting.
+- Run `pipenv run format` to format the code.
+- Run `pipenv run type_check` to run the static type checker ([mypy](https://github.com/python/mypy)).
 
+## Development using Docker
 
-## Usage 
+After making changes, run following commands.
 
-### Setup 
+- Run `docker-compose down` to stop the existing service.
+- Run `docker-compose up --build -d` to rebuild and start the container.
+
+Alternatively, you can just execute the `./rebuildDocker.sh` script.
```

Comment:
While changing the service, I find `uvicorn 'src.main:app' --reload --port=8082` to work very nicely compared to bothering with Docker which does not automatically apply your changes on save.